src/app/issues/[id]/author/page.tsx

```diff
@@ -0,0 +1,32 @@
+import Link from "next/link";
+import { ArrowLongLeftIcon } from "@heroicons/react/24/outline";
+import AuthorViewpointCard from "@/components/AuthorViewpoint/AuthorViewpointCard";
+import { Metadata } from "next";
+
+type AuthorViewPointProps = {
+    params: {
+        id: string;
+    };
+};
+
+export const metadata: Metadata = {
+    title: "CommonGround - 撰寫觀點",
+    keywords: "社會時事, 觀點, 理性討論, 撰寫觀點",
+};
+
+export default function AuthorViewPoint({ params }: AuthorViewPointProps) {
+    const id = params.id;
+
+    return (
+        <main className="mx-auto my-8 w-full max-w-7xl">
```

Comment:
Since this page doesn't have much content and is mostly client-side logic, I think we can consider making this page completely client-side rendered and move all login into `AuthorViewpointCard`.